cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -515,25 +570,31 @@ def act_sigmoid_scaled(x):
         500, activation="relu", kernel_regularizer=tf.keras.regularizers.l2(0.00005)
     )(x)
     x = Dropout(0.5)(x)
+    # specify element-wise activation
     output = Dense(1, activation=act_sigmoid_scaled)(x)
     model = tf.keras.Model([inputs[f] for f in all_cols], output)
+    # display the details of the Keras model
     model.summary()
 
     opt = tf.keras.optimizers.Adam(lr=hp.learning_rate, epsilon=1e-3)
 
-    # checkpoint callback to specify options for the returned Keras model
+    # checkpoint callback to specify the options for the returned Keras model
     ckpt_callback = BestModelCheckpoint(
         monitor="val_loss", mode="auto", save_freq="epoch"
     )
 
-    # Horovod: run training
+    # Horovod on Spark uses cloudpickle to send training function to workers for execution
+    # create an object of Store class
     store = Store.create(work_dir.remote_source)
+    # 'SparkBackend' uses `horovod.spark.run` to execute the distributed training function
+    # it returns a list of results by running 'train' on every worker in the cluster    
```

Comment:
If a comment is a continuation of the previous comment, let's have a comma at the end of the first comment, or begin the second and the following comments with uppercase.